test/dumbinputs.cpp

```diff
@@ -116,7 +117,7 @@ int main(int argc, char *argv[]) {
     return 1;
   }
   for (int k = 0; k < NN; ++k)
-    F[k] = sin((FLT)0.7 * k) + IMA * cos((FLT)0.3 * k); // set F for t2
+    F[k] = std::sin((FLT)0.7 * k) + IMA * std::cos((FLT)0.3 * k); // set F for t2
   ier = FINUFFT1D2(M, x, c, +1, 0, N, F, &opts);
   if (ier != FINUFFT_WARN_EPS_TOO_SMALL) {
     printf("1d2 tol=0:\twrong err code %d\n", ier);
```

Comment:
I'm assuming ``printf`` is fine here, thank god! Not sure why it wasn't fine earlier...